layouts/partials/navigation_level.html

```diff
@@ -28,32 +28,40 @@
         {{ $pageLocation := (path.Dir (path.Dir .File.Dir)) }}
         {{ if and (eq $parentDir $pageLocation) (not .Params.draft) }}
             {{ $desc := .Description }}
+            {{ $linktitle := .LinkTitle }}
+            {{ if isset .Params "status" }}
+                {{ $status := .Params.status }}
+                {{ if in $status "Experimental" }}
```

Comment:
@ChristinaMak I agree with @craigbox comments that we can use the the same symbol for any status - Experimental/Alpha/Beta (especially since we are including what it means in the description in the hover text). In that case, we can remove this check.